mappings/net/minecraft/item/BowItem.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_1753 net/minecraft/item/BowItem
+	FIELD field_30855 TICKS_PER_SECOND I
```

Comment:
Is this the only usage of `20` in this class? This looks like a weird usage, and I expect the actual usage to be more specific.

Reply:
Could this be related to the literal `20` in `BowAttackGoal#tick`? This might be the minimum number of ticks `x` that maximizes `BowItem#getPullProgress(x)`.

Reply:
Didn't see the `20` in `BowAttackGoal#tick`, but they mean probably the same thing. "ticks per second" is a bad name, do you think of any better?